fora22/DataStructure/Stack/foraStack_Linked_List/foraStackLinkedList.cpp

```diff
@@ -5,17 +5,17 @@
 #include "foraStackLinkedList.h"
 
 foraStackLinkedList::foraStackLinkedList() {
-    foraStackHead = nullptr;
-    foraStackTail = nullptr;
-    foraStackEndTail = this;
+    stackHead = nullptr;
+    stackTail = nullptr;
+    lastListTail = this;
```

Comment:
class 이름에 이미 stack이 들어 있으니까 변수 이름에 stack이 없어도 될 수 있습니다.
```suggestion
    head = nullptr;
    tail = nullptr;
    lastListTail = this;
```

Reply:
아 그렇네요 ptr 이 붙은 포인터 변수는 상관없을까요?

Reply:
뭐 ptr이나 p 접두어를 붙여서 pTail 이런 식도 표현하는데
이건 옛날 IDE가 안좋았던 시절에 쓰던 올드한 방식이고
요새는 context에 맞게만 해주면 되는 것 같습니다.

https://stackoverflow.com/questions/1722112/what-are-the-most-common-naming-conventions-in-c

Reply:
넵! 그럼 merge 하도록 하겠습니다.